llm_sandbox/kubernetes.py

```
@@ -376,6 +386,48 @@ def _reconfigure_with_pod_manifest(self) -> None:
+            # Verify pod exists and get its status
+            pod = self.client.read_namespaced_pod(name=pod_id, namespace=self.kube_namespace)
+            self._log(f"Connected to existing pod {pod_id}")
+            
+            # Check if pod is running
+            if pod.status.phase != "Running":
+                if pod.status.phase == "Pending":
+                    self._log(f"Pod {pod_id} is pending, waiting for it to start...")
+                    # Wait a bit for pod to start
+                    start_time = time.time()
+                    while time.time() - start_time < 60:  # Wait up to 1 minute
+                        pod = self.client.read_namespaced_pod(name=pod_id, namespace=self.kube_namespace)
```

Ruff (0.11.9)

Magic value used in comparison, consider replacing `60` with a constant variable

(PLR2004)